src/client/components/common/Card/index.tsx

```diff
@@ -12,9 +12,7 @@ const bigSize = css`
   min-height: 17.6rem;
 `;
 
-const smallSize = css`
-  min-height: 12.8rem;
-`;
+const smallSize = css``;
```

Comment:
why this? @pentcle

Reply:
i took off the fixed height as it was adding extra space at the base of the card container as such: 

<img width="981" alt="image" src="https://user-images.githubusercontent.com/80323528/135619418-d2810585-f960-4979-bb6e-121bbe339150.png">

Removing the fixed height allows the box to size to its contents eg

<img width="978" alt="image" src="https://user-images.githubusercontent.com/80323528/135619485-5a741c98-7743-40c6-a63a-3cea8d50e9ed.png">

Reply:
Yeah, the remaining space is intended in this case! Refer to figma